### PR TITLE
Change bech32 prefix to osmo

### LINKED ISSUE #126: Switch bech32 prefix to osmo instead of cosmos

### PR DESCRIPTION
**Pull request by sunnya97**
closes #126 

This also cleans up a lot of the testing infrastructure, in order to make this fix possible.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -813,34 +783,9 @@ google.golang.org/genproto v0.0.0-20201111145450-ac7456db90a6/go.mod h1:FWY/as6D
...
-google.golang.org/grpc v1.32.0/go.mod h1:N36X2cJ7JwdamYAgDz+s+rVMFjt3numwzf/HckM8pak=
-google.golang.org/grpc v1.33.1/go.mod h1:fr5YgcSWrqhRRxogOsw7RzIpsmvOZ6IcH4kBYTpR3n0=
-google.golang.org/grpc v1.33.2/go.mod h1:JMHMWHQWaTccqQQlmk3MJZS+GWXOdAesneDmEnv2fbc=
-google.golang.org/grpc v1.35.0 h1:TwIQcH3es+MojMVojxxfQ3l3OF2KzlRxML2xZq0kRo8=
-google.golang.org/grpc v1.35.0/go.mod h1:qjiiYl8FncCW8feJPdyg3v6XW24KsRHe+dy9BAGRRjU=
-google.golang.org/grpc v1.36.1 h1:cmUfbeGKnz9+2DD/UYsMQXeqbHZqZDs4eQwW0sFOpBY=
-google.golang.org/grpc v1.36.1/go.mod h1:qjiiYl8FncCW8feJPdyg3v6XW24KsRHe+dy9BAGRRjU=
 google.golang.org/grpc v1.33.2 h1:EQyQC3sa8M+p6Ulc8yy9SWSS2GVwyRc83gAbG8lrl4o=
 google.golang.org/grpc v1.33.2/go.mod h1:JMHMWHQWaTccqQQlmk3MJZS+GWXOdAesneDmEnv2fbc=
+google.golang.org/grpc v1.33.2/go.mod h1:JMHMWHQWaTccqQQlmk3MJZS+GWXOdAesneDmEnv2fbc=
```
Comment (ValarDragon) [inline go.sum:788]: Why did the later grpc versions get deleted, and an older one get added?

File: x/gamm/types/account_test.go [hunk 2]
```diff
@@ -10,6 +10,8 @@ import (
 	yaml "gopkg.in/yaml.v2"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	appparams "github.com/c-osmosis/osmosis/app/params"
```
Comment (ValarDragon) [inline x/gamm/types/account_test.go:14]: Lets rename this alias, too many lowercase p's in a row. Perhaps `appParams`?

Comment (ValarDragon) [inline x/gamm/types/account_test.go:14]:
```suggestion
	appParams "github.com/c-osmosis/osmosis/app/params"
```

File: x/gamm/types/account_test.go [hunk 3]
```diff
@@ -573,6 +575,7 @@ func TestPoolAccountPokeTokenWeights(t *testing.T) {
 }
 
 func TestPoolAccountMarshalYAML(t *testing.T) {
+	appparams.SetAddressPrefixes()
```
Comment (ValarDragon) [inline x/gamm/types/account_test.go:578]:
```suggestion
	appParams.SetAddressPrefixes()
```

File: x/gamm/types/msgs.go [hunk 4]
```diff
@@ -28,13 +28,17 @@ func ValidateFutureGovernor(governor string) error {
...
+
 	// validation for future owner
-	// "cosmos1fqlr98d45v5ysqgp6h56kpujcj4cvsjn6mkrwy"
+	// "osmo1fqlr98d45v5ysqgp6h56kpujcj4cvsjnjq9nck"
 	_, err := sdk.AccAddressFromBech32(governor)
 	if err == nil {
 		return nil
 	}
 
+	fmt.Println(err)
```
Comment (ValarDragon) [inline x/gamm/types/msgs.go:40]: Debug code

File: x/gamm/types/msgs_test.go [hunk 5]
```diff
@@ -8,12 +8,14 @@ import (
 
 	"github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	appparams "github.com/c-osmosis/osmosis/app/params"
```
Comment (ValarDragon) [inline x/gamm/types/msgs_test.go:12]:
```suggestion
	appParams "github.com/c-osmosis/osmosis/app/params"
```

File: x/gamm/types/msgs_test.go [hunk 6]
```diff
@@ -8,12 +8,14 @@ import (
...
 )
 
 func TestMsgCreatePool(t *testing.T) {
+	appparams.SetAddressPrefixes()
```
Comment (ValarDragon) [inline x/gamm/types/msgs_test.go:16]:
```suggestion
	appParams.SetAddressPrefixes()
```

File: x/gamm/types/msgs.go (hunk 4 above)
Comment (sunnya97) [reply x/gamm/types/msgs.go:40]:
```suggestion
```

**Review by ValarDragon — APPROVED** [review]
LGTM, assuming go.sum changes is from go mod tidy

**Pull request merged**